src/utils/vscodeUtils.ts

```diff
@@ -66,3 +66,13 @@ async function getUniqueFileName(folderPath: string, fileName: string, fileExten
 
     throw new Error('Could not find unique name for new file.');
 }
+
+export function fetchNodeModule(moduleName: string) {
```

Comment:
How about doing it relative to the current file's path?  Then you wouldn't need fallback code.  Also, you wouldn't have another exception which has to be ignored in the debugger.